frontend/src/components/window/workspace/Panel.svelte

```diff
@@ -100,7 +100,7 @@
 </script>
 
 <LayoutCol on:pointerdown={() => panelType && editor.handle.setActivePanel(panelType)} class={`panel ${className}`.trim()} {classes} style={styleName} {styles}>
-	<LayoutRow class="tab-bar" classes={{ "min-widths": tabMinWidths }}>
+	<LayoutRow class="tab-bar" classes={{ "min-widths": tabMinWidths }} on:dblclick={() => editor.handle.newDocumentDialog()}>
```

Comment:
If you add the click handler to a new element that fills the portion of the bar not occupied by tabs, you won’t need to stop event propagation. I think it feels a little cleaner.